### ISSUE TITLE
Feature request: Heat Capacitor

### ISSUE THREAD

**Issue by Vantive1** [ottopaulsen/node-red-contrib-power-saver#103]
Currently a stable electricity price, with buy/sell periods that has too low tradeValue will make the strategy node Heat Capacitor stabilize on the lower set point. My expectation from a comfort perspective is that it should stabilize on the set point. So the suggestion is that the strategy node provides a dT=0 for (longer) periods where the buy/sell actions are removed due to a trade value under the threshold or the electricity price is not volatile. (perhaps this complicates the trade algorithms...)

**Comment by TomTorger** [comment]
If the price/trade cost is slightly increasing, it will start with setpoint + dT (anticipating a coming maximum), but if it is decreasing, it will start with setpoint - dT (anticipating a coming minimum). 

The reason for the behavior is due to the situations when you lose your history. If we reboot our system will have to decide on whether to stay at T+dT or T-dT (or indeed T). In most cases, we will end up with the same pattern if we choose the algorithm above...

**Comment by Vantive1** [comment]
Ok, thanks for the explanation! I see the difficulty with making decisions without the history. 
But is the setpoint ever stabilized on dT=0? In what scenario? I have tried to test and figure that out but haven't succeeded...

**Comment by TomTorger** [comment]
Hehe... There is only one scenario where it actually ends up at dT=0, and that is when there is no price data at all. :-)

Otherwise, it will look at the gradient and put the setpoint at T+- dT.

**Comment by Vantive1** [comment]
I think it would be useful with some more documentation/explanation regarding how the algorithm is working and how the configuration is influencing the algorithm.

For example I'd like to be able understand where the switch between negative/positive dT occurs to be able to adjust those transition points in the high/low price cycle.

Btw thanks for the effort you are putting into this project! 🙏☀️

**Comment by TomTorger** [comment]
Yes, it is on my todo list :-) I hope to update the documentation so that it is a bit easier to set up and also to grasp where it is applicable, how it works and how to operate it. Its a bit of work though... For now, I'm prioritizing easier access to the schedule and documenting a dynamic setup in HA. But I'll get to the algorithm after that.
![image](https://user-images.githubusercontent.com/49571281/210453441-2caa940a-e9f8-41de-900a-2b86f8f39619.png)
